### DISCUSSION TITLE
Can't login to account, can't change password, can't contact support

### DISCUSSION THREAD

**Discussion by nijikokun** [npm/feedback#1040] [General]
Trying to reset my account's password. Every form I try and enter returns back an error stating that my account needs to have 2FA since it's high profile. But I can't turn on 2FA because I don't have access to the password manager I used for it anymore.

This is what I get when I try to submit any form on the site (including contact us):

![image](https://github.com/npm/feedback/assets/198276/44abbf00-37fa-4d12-bbd3-3987367b760c)

Can someone from npm please help?

**Comment by MylesBorins** [comment]
Can you try opening a ticket with GitHub support and share the ticket number, I'll get it forwarded to our team

**Reply by nijikokun** [reply]
Thanks @MylesBorins, just opened one, the ticket number is #2451632
